src/transformers/models/bart/modeling_tf_bart.py

```diff
@@ -136,7 +134,7 @@ def call(
             position_ids = tf.range(seq_len, delta=1, name="range")
             position_ids += past_key_values_length
 
-        return super().call(position_ids + self.offset)
+        return super().call(position_ids + tf.constant(self.offset, dtype=tf.int32))
```

Comment:
Should this be `dtype=position_ids.dtype` to avoid bugs if the user passes `int64` position IDs?